tests/Makefile

```diff
@@ -0,0 +1,12 @@
+INC=-I. -I../images/buffer -I../images -I../logging
```

Comment:
this is the makefile for tests. you'll need to edit this as you add more files for testing